tests/integration/devfile/cmd_devfile_delete_test.go

```diff
@@ -0,0 +1,86 @@
+package devfile
+
+import (
+	"github.com/openshift/odo/tests/helper"
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("odo devfile delete command tests", func() {
+	var namespace string
+	var context string
+	var currentWorkingDirectory string
+
+	// TODO: all oc commands in all devfile related test should get replaced by kubectl
+	// TODO: to goal is not to use "oc"
+	oc := helper.NewOcRunner("oc")
+
+	// This is run after every Spec (It)
+	var _ = BeforeEach(func() {
+		SetDefaultEventuallyTimeout(10 * time.Minute)
+		namespace = helper.CreateRandProject()
+		context = helper.CreateNewContext()
+		currentWorkingDirectory = helper.Getwd()
+		helper.Chdir(context)
+		os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "config.yaml"))
+
+		// Devfile commands require experimental mode to be set
+		helper.CmdShouldPass("odo", "preference", "set", "Experimental", "true")
```

Comment:
Smart!!!

Reply:
:D

Reply:
agreed +1

Reply:
@mik-dass if you dont mind, i shall make this change for the push integration tests when i am putting up a PR

Reply:
Yup, I like this and I already did that in my `odo create` PR, we should apply that to all integration tests under `tests/integration/devfile/`

Reply:
> @mik-dass if you dont mind, i shall make this change for the push integration tests when i am putting up a PR

Sure no problem